app/views/Dashboard/Overview/PercentageCardGroup/index.tsx

```diff
@@ -15,6 +15,7 @@ import {
     ContainerCard,
     Element,
     ListView,
+    Tooltip as DeepTooltip,
```

Comment:
Previously, in case of two tooltips, we cast Tooltip from recharts as ChartTooltip and keep the Tooltip from Deep UI as is. Maybe do the same here?